### ISSUE TITLE
How to get and show old value of field

### ISSUE THREAD

**Issue by phuongdm1987** [owen-it/laravel-auditing#38]
I can not get old value of field when show log by logCustomFields.

**Comment by anteriovieira** [comment]
Helo @phuongdm1987 ,

please, show me an example of what you like to do.

**Comment by phuongdm1987** [comment]
Hi @anteriovieira,

I want to declare $logCustomFields as below with aol value like {old.name}

public static $logCustomFields = [
        'name'  => 'The name change to {old.value} from {new.name}'
];

**Comment by anteriovieira** [comment]
Is correct? ```... {old.value} ...``` or would ```... {old.name} ...``` in your example

**Comment by anteriovieira** [comment]
Hi, Thank you for reporting. 
I will analyze and answer as soon as possible.  :wink:

**Comment by phuongdm1987** [comment]
Hi @anteriovieira,

when call  {new.name} only then ok
when call  {new.name} with {old.name} or {old.value} then fail, not show log.

**Comment by anteriovieira** [comment]
When it fails, it shows an error message? His log records contain old values?

**Comment by anteriovieira** [comment]
@phuongdm1987 , thank you for reporting. 

The failure is precisely because you do not have old value, to resolve use a default value for the field.

```php
public static $logCustomFields = [ 
    'name' => 'The name change to {old.value|Default value} from {new.name|Default value}' 
];
```

But you were right to report, this should not happen. I will adjust this as soon as possible

**Comment by anteriovieira** [comment]
Hi, @phuongdm1987 , Was adjusted, anything we are here.

New version available: `v2.2.2`
